### ISSUE TITLE
There is a critical issue in your code.

### ISSUE THREAD

**Issue by awesome1128** [eugenezadorin/airtable-php#1]
How are you?
Nice to meet you.
I found a critical issue in your code. You mentioned only >, !=, =, <, and so on in Query builder.
But for example, if I want to find data that contains a specific string from A column, then how can I build a Query?
Please let me know.
Thanks.

**Comment by eugenezadorin** [comment]
Hi. Could you please provide me some more details about your specific case?

Do you mean search by substring? Or this is about filtering rows by cell without knowing exact value of this cell?

Currently query builder has no methods to make such query in handy way, but you always can use raw formula.

If we talking about searching by substring, it can look something like that:

```php
// find records where My Column value starts with foo
$query->whereRaw("REGEX_MATCH({My Column}, '^foo')");
```

You can also use values from other cells in your formula:

```php
// find records where My Column contains value from Other Column
$query->whereRaw("REGEX_MATCH({My Column},('' & {Other Column} & ''))");
```

**Comment by awesome1128** [comment]
Thank you for your kind reply. I really appreciate it.
Yes, I wanted to find a record where column contains a substring. But I
think, this will be working the one you gave me. Will it be working for my
condition? Please let me know. Thanks. Regards
-------------------------------------------------------------------------------

You can also use values from other cells in your formula:

// find records where My Column contains value from Other Column
$query->whereRaw("REGEX_MATCH({My Column},('' & {Other Column} & ''))");

**Comment by eugenezadorin** [comment]
Sure, raw formula will work for you. Assume you look for users with name containing substring "Ivan". There are few options:

```php
$query->whereRaw("REGEX_MATCH({Username}, 'Ivan')"); // case-sensitive, any place in cell

$query->whereRaw("REGEX_MATCH({Username}, '^Ivan')"); // case-sensitive, cell value must start with Ivan

$query->whereRaw("REGEX_MATCH({Username}, '(?i)Ivan')"); // case-insensitive, any place in cell
```

You can read more about regexp syntax [here](https://github.com/google/re2/wiki/Syntax).

I think I could provide some convenient query builder methods for this later. Something like that:

```php
$query->whereLike("Username", "Ivan");
$query->where("Username", "like", "Ivan%");
$query->whereMatch("Username", "(?i)ivan");
```

**Comment by awesome1128** [comment]
It sounds good!
Yes, if you provide some convenient query builder methods , then it will be
great!
I really love to use your code.
Ill be waiting for your updates.
Thank you so much.

**Comment by eugenezadorin** [comment]
Implemented in v0.3.0

Few examples:

```php
// look for emails, matching @gmail.com in case-insensitive way
$query->where('email', 'match', '(?i)^(.+)@gmail.com$');

// look for names, which starts with Ivan (case-sensitive!)
$query->where('name', 'like', 'Ivan%');
```
